### DISCUSSION TITLE
🙏 Exclude worker.js files when using  CDN

### DISCUSSION THREAD

**Discussion by yjhtry** [vercel/next.js#51381] [Help] [answered]
### Summary

After configuring`assetPrefix`, all js files in my project will be loaded through CDN, which is no problem. But`new Woker('file path')` must require the same source origin as the current website

So how can I exclude woker.js file from CND request

**Comment by raym0nd93** [comment]
Any update on this?

**Reply by yjhtry** [reply, marked as answer]
I solved the problem this way [remote-web-worker](https://github.com/jantimon/remote-web-worker)
